pkg/cortex/cortex.go

```diff
@@ -329,3 +389,19 @@ func orderedDeps(m moduleName) []moduleName {
 	}
 	return result
 }
+
+// find modules in the supplied list, that depend on mod
+func findReverseDeps(mod moduleName, mods []moduleName) []moduleName {
```

Comment:
I find the function name `findReverseDeps` a bit misleading. Before looking at its, I thought it was returning the results in reverse order.

Reply:
Would "Inverse" make more sense?

Reply:
I use `findInverseDeps` now. I'm open to suggestions.